arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/ParTupledN.kt

```diff
@@ -137,3 +138,78 @@ suspend inline fun <A, B, C> parTupledN(
   crossinline fc: suspend () -> C
 ): Triple<A, B, C> =
   parMapN(ctx, fa, fb, fc, ::Triple)
+
+/**
+ * Runs [fa], [fb], [fc], [fd] in parallel on [Dispatchers.Default] and combines their results into [Tuple4].
+ *
+ * ```kotlin:ank:playground
+ * import arrow.fx.coroutines.*
+ *
+ * suspend fun main(): Unit {
+ *   //sampleStart
+ *   val (a, b, c, d) = parTupledN(
+ *     { "First one is on ${Thread.currentThread().name}" },
+ *     { "Second one is on ${Thread.currentThread().name}" },
+ *     { "Third one is on ${Thread.currentThread().name}" },
+ *     { "Fourth one is on ${Thread.currentThread().name}" }
+ *   )
+ *   //sampleEnd
+ *  println("$a\n$b\n$c\n$d")
+ * }
+ * ```
+ *
+ * @param fa value to parallel map
+ * @param fb value to parallel map
+ * @param fc value to parallel map
+ * @param fd value to parallel map
+ *
+ * @see parMapN for a function that can run on any [CoroutineContext].
+ */
+suspend inline fun <A, B, C, D> parTupledN(
+  crossinline fa: suspend () -> A,
+  crossinline fb: suspend () -> B,
+  crossinline fc: suspend () -> C,
+  crossinline fd: suspend () -> D,
+): Tuple4<A, B, C, D> =
+  parTupledN(Dispatchers.Default, fa, fb, fc, fd)
+
+/**
+ * Runs [fa], [fb], [fc], [fd] in parallel on [ctx] and combines their results using the provided function.
```

Comment:
I guess is a copy-paste nit from the docs above, but there's no function whatsoever, so this might be Tuple4

```suggestion
 * Runs [fa], [fb], [fc], [fd] in parallel on [ctx] and combines their results into [Tuple4].
```